bCNC/lib/bpath.py

```diff
@@ -823,18 +846,18 @@ def distPoint(self, dist):
 				return segment.distPoint(dist)
 			dist -= segment.length()
 
-        #----------------------------------------------------------------------
-        # Return linearized path (arcs are subdivided to lines)
-        #----------------------------------------------------------------------
+		#----------------------------------------------------------------------
+		# Return linearized path (arcs are subdivided to lines)
+		#----------------------------------------------------------------------
```

Comment:
Why use two level indentation comments for methods with single level of indentation? (This is in multiple places)